src/modules/service-catalog/ServiceCatalogList.tsx

```diff
@@ -0,0 +1,102 @@
+import { useCallback, useEffect, useState } from "react";
+import ServiceCatalogListItem from "./components/service-catalog-list-item/ServiceCatalogListItem";
+import type { ServiceCatalogItem } from "./data-types/ServiceCatalogItem";
+import { Col, Row } from "@zendeskgarden/react-grid";
+import styled from "styled-components";
+import { useTranslation } from "react-i18next";
+
+import { CursorPagination } from "@zendeskgarden/react-pagination";
+
+const StyledCol = styled(Col)`
+  margin-top: ${(props) => props.theme.space.sm};
+  flex: 0 0 25%; /* Set width to 25% for 4 columns */
+  max-width: 25%; /* Set max-width to 25% for 4 columns */
+  padding: 0 24px 24px 0;
+`;
+
+type Meta = {
+  before_cursor: string;
+  after_cursor: string;
+  has_more: boolean;
+};
+
+export function ServiceCatalogList() {
+  const [serviceCatalogItems, setServiceCatalogItems] = useState([]);
+  const [meta, setMeta] = useState<Meta | null>(null);
+  const [currentCursor, setCurrentCursor] = useState<string | null>(null);
+  const { t } = useTranslation();
+  const fetchServiceCatalogItems = useCallback(async (cursorParam) => {
+    try {
+      console.log("cursorParam", cursorParam);
+      const response = await fetch(
+        cursorParam
+          ? `/api/v2/custom_objects/service_catalog_item/records?page[size]=16&${cursorParam}`
+          : `/api/v2/custom_objects/service_catalog_item/records?page[size]=16`
+      );
+      const data = await response.json();
+      if (response.ok) {
+        const records = data.custom_object_records.map(
+          ({
+            id,
+            name,
+            custom_object_fields,
+          }: {
+            id: number;
+            name: string;
+            custom_object_fields: { description: string };
+          }) => ({ id, name, description: custom_object_fields.description })
+        );
+        setMeta(data.meta);
+        setServiceCatalogItems(records);
+      }
+    } catch (error) {
+      console.error("Error fetching service catalog items:", error);
+    }
+  }, []);
+
+  const onNext = () => {
+    if (meta && meta.after_cursor) {
+      setCurrentCursor(`page[after]=${meta.after_cursor}`);
+    }
+  };
+
+  const onPrevious = () => {
+    if (meta && meta.before_cursor) {
+      setCurrentCursor(`page[before]=${meta?.before_cursor}`);
+    }
+  };
+
+  useEffect(() => {
+    fetchServiceCatalogItems(currentCursor);
+  }, [currentCursor, fetchServiceCatalogItems]);
+
+  return (
+    <div>
+      <Row>
```

Comment:
From a semantic and a11y point of view, I think we should render the items as an unstyled unordered list. I don't know if it feasible with Garden Grid since Row and Col are divs

Reply:
It is mostly a nitpick. For example, if you look at the [template for categories ](https://github.com/zendesk/copenhagen_theme/blob/e7fcde18dcf31608ba4ae16602e3f3e507e52aae/templates/home_page.hbs#L17) in the homepage, you can see that even if is rendered as a grid of elements the markup is a unordered list.

Probably it is better for screen readers to announce this grid as a list of items as well, but I am not 100% sure of that. We can keep the `div`s for now and let's see when we will have a a11y review if we need to change that